.travis.yml

```diff
@@ -22,7 +22,8 @@ before_install:
   - wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar --output-document=php-cs-fixer.phar
 
 install:
-  - composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
+  - composer global show hirak/prestissimo -q || travis_retry composer global require hirak/prestissimo --prefer-dist --no-interaction --no-progress
```

Comment:
Please add some linebreaks here.

Reply:
like between this and next `install` steps, or to split this line? not sure what you mean to be honest.
well, in the end I left allowance to modify my PR to you guys, so feel free to add them yourself however you want to have them!

Reply:
Just to split this line, but yeah I can amend it.